### ISSUE TITLE
Lower fees for stable coin transfers

### ISSUE THREAD

**Issue by Henry-E** [project-serum/serum-dex#135]
Something which came up in some other discussions. Is there any reason for serum to charge high bps fees for trades between "stable" coins like usdc and usdt.

The changes needed to make this happen might have further reaching consequences but maybe there could be some governance mechanism for approving lower fees for these kinds of integral trading pairs.

**Comment by armaniferrante** [comment]
Agreed that 1) all such parameters should be configurable by an authority and 2) the authority should be a dao program.

**Comment by secretshardul** [comment]
Quantifying, the base fee tier (0.22%) is 440% more than Uniswap v3 (0.05%). Even when 1 million serum tokens are staked (0.1%), it's 100% higher.

**Comment by keone** [comment]
this was altered; this ticket can probably be closed
